diff --git a/node_modules/expo/src/Expo.fx.tsx b/node_modules/expo/src/Expo.fx.tsx
index b54d1b1..8eafbe9 100644
--- a/node_modules/expo/src/Expo.fx.tsx
+++ b/node_modules/expo/src/Expo.fx.tsx
@@ -18,22 +18,7 @@ if (isRunningInExpoGo()) {
 // Warn if the New Architecture is not explicitly enabled in the app config and we are running in Expo Go.
 // This could be problematic because you will be developing your app with the New Architecture enabled and
 // but your builds will have the New Architecture disabled.
-if (__DEV__ && isRunningInExpoGo() && process.env.NODE_ENV === 'development') {
-  ['android', 'ios'].forEach((platform) => {
-    if (
-      Platform.OS === platform &&
-      Constants.expoConfig?.[platform]?.newArchEnabled !== true &&
-      Constants.expoConfig?.newArchEnabled !== true
-    ) {
-      // Wrap it in rAF to show the warning after the React Native DevTools message
-      requestAnimationFrame(() => {
-        console.warn(
-          `🚨 React Native's New Architecture is always enabled in Expo Go, but it is not explicitly enabled your project app config. This may lead to unexpected behavior when you create a production or development build. Set "newArchEnabled": true in your app.json.\nLearn more: https://docs.expo.dev/guides/new-architecture/`
-        );
-      });
-    }
-  });
-}
+
 
 // Disable the "Open debugger to view warnings" React Native DevTools warning in
 // Expo Go and expo-dev-client, because launching the debugger from there will not
